src/routes/endpoints/v1/action-type/list-action-types.ts

```diff
@@ -19,9 +19,8 @@ const routes: FastifyPluginAsync = async fastify => {
     }),
     async (req, res) => {
       const { actionType } = fastify.sql;
-      return EitherAsync(() =>
-        fastify.db.any(actionType.findAllUserActions()),
-      ).caseOf({
+      return actionType.findAllUserActions(fastify.db)
```

Comment:
Getting an error here; `Expected 2 arguments, but got 1.`

Reply:
Are you sure it's in the PR and not because you tested the `void` => `never` suggestion you made earlier? 🤔